workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -143,28 +143,6 @@ def plot_1d_with_region_bar(
     return HVPlot(_set_opts_and_overlay(hmap))
 
 
-def plot_2d(
```

Comment:
this function was not used anywhere except in a unit test. Deleting since we don't want to encourage 2D holoviews plots IMO.